microshift_running_apps/microshift-operators.adoc

```diff
@@ -10,9 +10,67 @@ You can use Operators with {microshift-short} to create applications that monito
+//TODO: see OSDOCS-8989 for summary and xref after PR is merged
+
+[id="microshift-olm"]
+== Using Operator Lifecyle Manager
+OLM is used in {microshift-short} for installing and running optional link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/architecture/control-plane#olm-operators_control-plane[add-on Operators].
+
+* Cluster Operators as applied in {ocp} are not used in {microshift-short}.
+* You must use the CLI to conduct OLM activities with {microshift-short}. The console and OperatorHub GUIs are not available, but you can take all the same actions with the CLI.
+
+[IMPORTANT]
+====
+Before using an Operator, verify with the provider whether the Operator is supported on {product-title}.
+====
+
+[id="microshift-file-based-olm-catalogs"]
+== File-based Operator catalogs
+
+To run {microshift-short} with your own curated catalogs and OLM, use the file-based catalog structure. For a complete explanation of the file-based structure, see the "Additional resources" section that immediately follows this one.
+
+include::modules/olm-fb-catalogs-example.adoc[leveloffset=+2]
+include::modules/olm-fb-catalogs-guidelines.adoc[leveloffset=+2]
+include::modules/olm-creating-fb-catalog-image.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html/cli_tools/opm-cli[`opm` CLI reference]
+* link:https://https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/operators/understanding-operators#olm-file-based-catalogs_olm-packaging-format[File-based catalogs]
+
+include::modules/olm-creating-catalog-from-index.adoc[leveloffset=+2]
+
+[id="microshift-using-olm-disconnected"]
+== Using OLM in disconnected clusters
+For {microshift-short} clusters that are installed on restricted networks, also known as _disconnected clusters_, Operator Lifecycle Manager (OLM) by default cannot access the Red Hat-provided OperatorHub sources hosted on remote registries because those remote sources require full internet connectivity.
+
+You can enable your cluster to use OLM in a restricted network using a workstation that has full internet access. The workstation is used to prepare local mirrors of the remote sources, and push the content to a mirror registry. See xref:../microshift_install/microshift-deploy-with-mirror-registry.adoc#microshift-deploy-with-mirror-registry[Mirroring container images for disconnected installations].
+
+The following process is required to enable OLM in restricted networks:
+
+* Disable the default remote OperatorHub sources for OLM.
+* Use a workstation with full internet access to create and push local mirrors of the catalog content to a mirror registry.
+* Configure OLM to install and manage Operators from local sources on the mirror registry instead of the default remote sources.
```

Comment:
I'm not sure if there's a need to configure OLM - @kevinrizza WDYT?

I'd say user needs to:
- provide a reference in CatalogSource that is accessible
- and/or configure CRI-O to use the mirror instead of original registry (which should be part of general configuration for disconnected env already)

Reply:
took straight from the olm docs, https://docs.openshift.com/container-platform/4.14/operators/admin/olm-restricted-networks.html so I guess it depends on what we need to do for microshift?

Reply:
Configure OLM in this nomenclature means "add a CatalogSource to the cluster, which allows OLM's catalog operator to use the catalog for content"

Reply:
Gotcha, I interpreted it that we need to make OLM itself aware of the offline, but it's "Catalog with reachable image" instead

Reply:
I'll update to just say that directly.